crates/feedback/src/system_specs.rs

```diff
@@ -29,6 +30,12 @@ impl SystemSpecs {
             .app_metadata()
             .os_version
             .map(|os_version| os_version.to_string());
+        let detail = match ReleaseChannel::global(cx) {
```

Comment:
Instead of reading `ReleaseChannel::global` here again, we might want to store the full `ReleaseChannel` in the `release_channel` binding so we can reuse it here, and then only call `.display_name()` on it when we go to construct the `SystemSpecs` down below.